templates/swags.me/trackingStatus.html

```diff
@@ -0,0 +1,328 @@
+                    >
+                      YOUR ORDER IS ON THE WAY
+                    </h2>
+                    <table
+                      width="100%"
+                      cellpadding="0"
+                      cellspacing="0"
+                      role="presentation"
+                    >
+                      <tbody>
+                        <tr>
+                          <td style="background: rgba(217, 217, 217, 1); padding: 8px 16px; display: flex; flex-direction: column;">
+                            <p style="font-size: 18px; line-height: 1; font-weight: normal; color: rgb(0, 0, 0); font-family: Inter;">Tracking Number</p>
+                            <p style="font-size: 18px; line-height: 1; font-weight: 600; color: rgb(0, 0, 0); font-family: Inter;">{{trackingNo}}</p>
+                          </td>
+                        </tr>
+                      </tbody>
+                    </table>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+
+            <table
+              width="100%"
+              cellpadding="0"
+              cellspacing="0"
+              role="presentation"
+              style="margin: 16px 0;"
+            >
+              <tbody>
+                <tr>
+                  <td
+                    style="border:1px solid rgb(19, 84, 71); padding: 18px 32px; width: 100%; text-align: center;"
+                  >
+                    <a
+                      href="{{trackPackage}}"
+                      target="_blank"
+                      style="display: block; width: 100%; color: rgb(19, 84, 71); text-decoration: none; font-size: 20px; line-height: 1.2; font-weight: bold; margin: 0 auto;"
+                      >Track Package</a
+                    >
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+
+            <table
+              width="100%"
+              cellpadding="0"
+              cellspacing="0"
+              role="presentation"
+              style="padding-bottom: 32px;"
+            >
+              <tbody>
+                <tr>
+                  <td style="text-align: center; padding: 8px 16px 32px 16px;">
+                    <p
+                      style="font-size: 18px; line-height: 1.6; color: rgba(52, 64, 84, 1); margin: 0 auto; text-align: center;"
+                    >
+                      Your order's is on its way. Use the link above to track its progress.
+                    </p>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+
+            <table
+              width="100%"
+              cellpadding="0"
+              cellspacing="0"
+              role="presentation"
+            >
+              <tbody>
+                <tr>
+                  <td style="display: flex; justify-content: space-between; padding: 24px 0 0 0; border-top: 2px solid rgb(234, 236, 240);"">
```

Comment:
remove the extra quote at the end
```suggestion
                  <td style="display: flex; justify-content: space-between; padding: 24px 0 0 0; border-top: 2px solid rgb(234, 236, 240);">
```

Reply:
1. Yes I double checked the padding and spacing of the marked items it is 24px. So I changed it accordingly.
<img width="369" height="748" alt="image" src="https://github.com/user-attachments/assets/6d1adee0-c57e-4f4d-8633-6dc0872b68bb" />

2. I tried sending it via test email in the desktop view and mobile view it is not breaking.
<img width="396" height="775" alt="image" src="https://github.com/user-attachments/assets/591e28f8-467f-43da-99a7-26b140f0dc26" />

3. I removed flexbox because it was breaking the layout and changed it to the table based layout for broader email client support.